docs/guides/soloproject/soloproject.md

```diff
@@ -44,10 +50,11 @@ intermediate (Tier 2), or experienced (Tier 3).
 <tr>
 <td>Tier&nbsp;1</td>
 <td>
-<p>This tier is for developers who are just starting out in web development.</p>
+<p>This tier is for developers who are starting out in frontend web development.</p>
 <strong>Skills:</strong>
 <ul>
-    <li>Basic understanding of CSS and HTML.</li>
+    <li>Basic understanding of CSS and HTML. You may use also CSS processors like SASS, LESS, Stylus, TailwindCSS, etc.</li>
+    <li>Ability to create simple apps using any frontend framework such as ReactJS, SvelteJS, VueJS, Angular, Django, etc.
```

Comment:
Django is not a frontend framework.

Reply:
@cherylli This is true. I included it so there would be at least one Framework suitable for Python developers. Is there another one that would be better to use?

Reply:
no, python is just not a frontend thing

Reply:
Thanks! Updated to remove Django